strawberryfields/backends/tfbackend/circuit.py

```diff
@@ -684,8 +647,7 @@ def measure_homodyne(self, phi, mode, select=None, **kwargs):
                 phi (float): phase angle of quadrature to measure
                 mode (int): which mode to measure.
                 select (float): user-specified measurement value (used instead of random sampling)
-                **kwargs: can be used to pass a session or a feed_dict. Otherwise a temporary session
-                and no feed_dict will be used.
+                **kwargs:
 
         Returns:
             The measured value (or a list of measured values when running in batch mode).
```

Comment:
It'd be nice if the incomplete/wrong docstrings would be fixed when each method is updated, e.g. the Returns: statement should be of the format `type: explanation`